astropy/io/ascii/core.py

```diff
@@ -30,6 +30,8 @@
 from ...utils.data import get_readable_fileobj
 from . import connect
 
+from . import core
```

Comment:
Don't import yourself.

Reply:
Oh, can you say how I should be calling `InconsistentTableError`?

Reply:
I've removed the import; sorry, I didnt realize I was importing `core` in `core.py`.